pkg/didcomm/protocol/introduce/service.go

```diff
@@ -130,7 +163,8 @@ func (s *Service) processFailure(id string, pErr error) error {
 }
 
 // abandon updates the state to abandoned and trigger failure event.
-func (s *Service) abandon(thID string, msg *service.DIDCommMsg, _ error) error {
+func (s *Service) abandon(thID string, msg *service.DIDCommMsg, pErr error) error {
+	logger.Infof("go to abandon: %v", pErr)
```

Comment:
i assume you are cleaning up log levels in a future PR?

Reply:
removed!

Reply:
It was a general comment :). There are several logger lines that should not "INFO" level.